src/Api/Controllers/TwoFactorController.cs

```diff
@@ -351,7 +351,7 @@ public async Task<TwoFactorProviderResponseModel> PutDisable([FromBody]TwoFactor
         [HttpPost("get-recover")]
         public async Task<TwoFactorRecoverResponseModel> GetRecover([FromBody]TwoFactorRequestModel model)
         {
-            var user = await CheckAsync(model.MasterPasswordHash, false);
+            var user = await CheckAsync(model.Secret, false);
```

Comment:
`CheckAsync` is local & widely used throughout this controller. Could we pass an object to it instead of this string value to make sure we don't have to refactor each one of these calls in the future when we inevitably need to check something else? May need to make an interface, but I think that would refactor and expand a lot nicer in the future.